### DISCUSSION TITLE
Can't dismiss event reminders

### DISCUSSION THREAD

**Discussion by GerardEarley** [kewisch/gdata-provider#708] [Support]
No matter what I do I can't dismiss any of the event reminders that pop up from Provider.

I'm running Thunderbird 115.6.0 (64-bit) on Windows 11 (Although the same problem also occurred on 10 Pro) and Provider 115.0.2

![Reminders](https://github.com/kewisch/gdata-provider/assets/10418107/c9b0d346-94a4-4143-a445-b2cebb7e94b4)

**Comment by maxwell-k** [comment]
I think I hit the same issue. I [enabled debugging](https://github.com/kewisch/gdata-provider/wiki#enabling-debugging)  then opened the Error Console and then I could see an error message saying effectively: "I'm not the owner I can't update the event". As a workaround I went into the events in Google Calendar and removed the notification for events that are in the past.

**Comment by carlesm** [comment]
Same thing happens to me, but how do you @maxwell-k  disable notifications for past events? Going to those events one by one?

**Reply by maxwell-k** [reply]
Yes. In my case there were approximately then past events that I could not dismiss in the user interface. They were all relatively recent, from the last six months.

I don't think this is a solution; it is just a workaround.

I wonder should we open an issue?

**Comment by megodoonch** [comment]
I have the same problem. Indeed, they all do seem to be past repeated events. I also have a "!" sign next to the account name and when I hover it says "The calendar <redacted> is momentarily not available", and yet I can create events on Thunderbird that then appear on my google calendar. Do you have these effects too?

I only just started using Thunderbird again, so I don't know what triggered this trouble for me.

I'll try getting rid of the notifications in the old events (for some reason it's not that long a list) but I think this is a bug.

**Comment by mikeyjoel** [comment]
Same issue on my end using version 115.8.1 on opensuse tumbleweed from the official repositories (not flatpak/snap).

**Comment by stevepiercy** [comment]
I have the same problem, and could not find another open issue in the tracker. I created a new bug report in the issue tracker. https://github.com/kewisch/gdata-provider/issues/781

**Comment by JPMHH** [comment]
Same issue for me, in all versions, on different OS, for years now, since I use this nice addon.
All events which are automatically created by an invitation (E-Mail) can't be dismissed. 
This is very anoying and I have to delete those events. If it's a permission problem, as others wrote, that the event is not owned by me, why can I delete it but not dismiss? This is illogical for me.

This issue is known for years. Why is this obviously ignored? Will this issue ever be solved?

**Reply by GerardEarley** [reply]
So the only way to stop the issue occurring is to disable "Events from Gmail" from creating events or set the events to be created in a calendar that is NOT displayed in Thunderbird.

https://support.google.com/calendar/answer/6084018?hl=en-GB

**Comment by victor2112v** [comment]
Same problem, las versions of Thunderbird and Provider Google Calendar. 
When I have only in 1 PC login, everything is ok, but when I try to login in both of my computers and work with it, reminders freeze and doesn´t work properly.

**Comment by toiletbowlkuku** [comment]
I am also getting an error dismissing notifications from gmail generated events.
```
Status Code: 2147500037, The request cannot be processed.

Server Replied with 400
If you're seeing this message after snoozing or dismissing a reminder and this is for a calendar you do not want to add or edit events for, you can mark this calendar as read-only to avoid such experience in future. To do so, get to the calendar properties by right-clicking on this calendar in the list in the calendar or task view.
```

**Comment by clari7744** [comment]
I'm having this issue in Thunderbird 128 on Ubuntu 24.04. As @megodoonch said 
>  I also have a "!" sign next to the account name and when I hover it says "The calendar is momentarily not available"
I get this as well when I try to dismiss the reminders

**Comment by mhalano** [comment]
Same here. I had to disable missed reminders for a while, so I can't have this problem.

**Comment by higgsja** [comment]
I went into google calendar and deleted the event there. That seems to have done it for me.

**Comment by PDM996** [comment]
I have removed and reimported the calendar, after having deleted the event/s that created the issue, and I have not had the massage popping up again...so far

**Comment by kewisch** [comment]
This issue will I believe eventually show up again on the release version of the Provider. You have two options:

1) Disable the "Show Missed" option in Thunderbird
2) Install the beta https://kewisch.github.io/gdata-provider/beta.html

Option 1 might not fully resolve the issue, but will make it a bit less painful.
